test/cpp/test_xla_sharding.cpp

```diff
@@ -435,5 +435,17 @@ TEST_F(XLAShardingTest, PrepareOutputShardingPropagation) {
           ->HasValue());
 }
 
+TEST_F(XLAShardingTest, TestForiLoopAddUnusedParameterInXlaComputation) {
+  xla::Shape shape = xla::ShapeUtil::MakeShape(xla::PrimitiveType::F32, {});
+  // Build simple addition.
+  xla::XlaBuilder b("builder");
+  auto x = xla::Parameter(&b, /*parameter_number=*/0, shape, "p0");
+  xla::Shape shape2 = xla::ShapeUtil::MakeShape(xla::PrimitiveType::F32, {});
+  auto zzz = xla::Parameter(&b, /*parameter_number=*/1, shape2, "p1");
+  auto y = xla::Add(x, xla::ConstantR0<float>(&b, 3));
+  xla::XlaComputation xla_computation =
+      ConsumeValue(b.Build(/*remove_dynamic_dimensions=*/false));
```

Comment:
q: what is this test testing? should there be an assert?

Reply:
this test is to confirm unused paramater created by `xla::Parameter` would be included in generated XlaComputation from `Build`

Thanks, added `EXPECT_EQ` to assert generated XlaComputation has included unused paramater

Reply:
We don't need this test, since there are lots unit tests in xla for this.

Reply:
Thanks, deleted